cli/help.md

```diff
@@ -49,6 +49,7 @@ Basic options:
 --preserveSymlinks          Do not follow symlinks when resolving files
 --shimMissingExports        Create shim variables for missing exports
 --silent                    Don't print warnings
+--failAfterWarnings         Exit with an error code if there was a warning during the build
```

Comment:
We also need to extend documentation in the `docs` folder,  just check where and how e.g. `--enivronment` is documented.